app/src/main/java/com/xmartlabs/daydreaming/ui/LoginRegisterFragment.java

```diff
@@ -86,16 +89,29 @@ void onClickedLogin() {
 
   @OnClick(R.id.create_login_textview)
   void onClickedCreateLogin() {
+    setupErrorFieldsAsInvisible();
     //TODO create user and login user
     if (isRegisterSelected) {
       if (passwordFieldView.getText().toString().length() < 5) {
```

Comment:
Doesn't matter. At this point this code is going to production, so it should be held to the same standards as the rest